clickable.yaml

```diff
@@ -1,47 +1,40 @@
 clickable_minimum_required: '7'
 builder: go
+prebuild: mkdir -p ${BUILD_DIR}/axolotl-web && cp ${ROOT}/axolotl-web/dist/* ${BUILD_DIR}/axolotl-web -R
 postbuild: ${ROOT}/scripts/postbuild.sh ${INSTALL_DIR};
```

Comment:
Do we need this $ROOT variable? Could we instead use `src_dir` also for the `postbuild` step to avoid relying on external variables?

Reply:
Yes, `${SRC_DIR}` would be more logical and it currently (by default) is the same as `${ROOT)`.

Reply:
The clickable.json is in the root folder so root and `$SRC_DIR` are the same. We can swith it.

Reply:
Funny, i see now your responses jonnius, we must have written the answers in the same moments.